src/coveralls.jl

```diff
@@ -158,7 +166,11 @@ module Coveralls
     git_info can be either a `Dict` or a function that returns a `Dict`.
     """
     function submit_token(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
-        data = Dict("repo_token" => ENV["REPO_TOKEN"],
+        data = Dict("repo_token" => get(ENV,"COVERALLS_TOKEN") do
+                            get(ENV, "REPO_TOKEN") do #backward compatibility
+                                error("Coveralls submission from AppVeyor requires a COVERALLS_TOKEN environment variable")
```

Comment:
The "AppVeyor" here should be removed. 

Also (and I realize this is not your fault, it's in the existing code), I wonder why repo token is not supported for Travis. According to the Travis documentation, it may also be needed on Travis, namely for private repos.

The reason I bring this up is that this might be an opportunity to refactor this code.